Update interfaces/webgl1.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/d952a67/whatwg/idl/webgl1.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/469883037

foolip commented:
This has introduced a failing subtest called "WebGLRenderingContext interface: member canvas" in the webgl1-idlharness.any.html test, expecting that 'The prototype object must not have a property "canvas"'. That must be because of `[Exposed=Worker] readonly attribute OffscreenCanvas canvas`.

Need to check if having two attributes by the same name is valid per Web IDL.

autofoolip committed:
Update interfaces/webgl1.idl

Source: https://github.com/tidoust/reffy-reports/blob/d952a67/whatwg/idl/webgl1.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/469883037

foolip commented:
I'll close this so that we get a new PR with fresh results.
